src/server/common/dbconn.cpp

```diff
@@ -878,6 +882,10 @@ mariadb::statement_ref dbconn::query(const mariadb::connection_ref& connection,
 		{
 			stmt->set_string(i, std::get<const char*>(param));
 		}
+		else
+		{
+			throw mariadb::exception::base("Unsupported parameter type in SQL query at position " + std::to_string(i));
```

Comment:
Is just the position going to be enough to know what this is about? I'd at least include `sql`.

Reply:
I'm not sure what that would accomplish? The issue in this case is that one of the types that `sql_parameters` could be wasn't handled, and putting a static_assert in the else block wouldn't work.

Reply:
The point of a static assert would be to get a compile error instead of a runtime error. I don't mean the assert would go in the else. Need to check how the assert would best be written. If there is no good way for an assert at least comments that this if chain needs to be in sync with the variant type from `sql_parameters` would be some help to avoid this issue in the future.

Reply:
Added the comment.